### ISSUE TITLE
Improve link styles (a11y)

### ISSUE THREAD

**Issue by aviannamiller** [omeka/theme-rhythm#19]
Link to body text contrast should be at least 3:1, it's currently 2:1 for Rhythm and Blue and 1.5:1 for Fall Rhythm (also problematic on hover for both), 2.1 on hover for Spring Rhythm.

**Comment by aviannamiller** [comment]
For the tag browse view:
- Spring Rhythm, secondary nav links pick up `.spring .tags a:hover {color: #ccdd44;}` on hover (1.5:1)
- In the tag cloud, tags are not visible at all for Spring Rhythm or Rhythm and Blue as they are styled in white text on a white background. For Fall Rhythm, it's white text on `rgb(210, 171, 64)` (only 2.2:1).
